### DISCUSSION TITLE
Staging Part 3 changes be like...

### DISCUSSION THREAD

**Discussion by NexRX** [palexdev/MaterialFX#37] [Archived]
<img src="http://pm1.narvii.com/6621/8031f801f8b8dcddf5cbce0fbdc5e5d0fc97c07d_00.jpg">

**Comment by palexdev** [comment]
Hahaha thanks, the next version is almost there
There are still little changes I want to do, and then I'll bump the version and publish it 😁

**Reply by NexRX** [reply]
You take all the time you need mate. We all know what is like when things are rushed. The more little changes the prouder you can be after all <3
